spec/migrations/20230709181609_drop_archived_container_projects_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_migration
+
+describe DropArchivedContainerProjects do
+  let(:container_group_stub) { migration_stub(:ContainerGroup) }
+
+  migration_context :up do
+    it "reassigns container_project_ids"
+      untouched = container_group_stub.create!(:container_project_id => 1)
+      touched1  = container_group_stub.create!(:container_project_id => 3, :old_container_project_id => 3, :deleted_on => Time.now.utc)
+      touched2  = container_group_stub.create!(:container_project_id => nil, :old_container_project_id => 4, :deleted_on => Time.now.utc)
```

Comment:
Probably doesn't matter, but could you create container_project records for these allowing the ID to be created in the right region versus hard-coded IDs?